articles/multicollinearity/index.md

```diff
@@ -0,0 +1,131 @@
+
+Collinearity refers to a condition whereby we have a pair of heavily correlated features to each other in data. For a variable to be considered an independent variable, it has to be distinct from other variables. It has to be independent of other variables. This means that we should not have the ability to derive this variable's values by using other independent variables. Consider another equation of a dependent variable concerning independent variables below.
+
+$$ y = c + m_1x_1 + m_2x_2 + m_3x_3 + … + m_nx_n $$
+
+Say $x_1$ represents total exam marks. $x_2$ represents exam one. $x_3$ is exam two. To get the value of $x_1$, we can add $x_2$ and $x_3$. This shows that $x_1$ is not independent. The implication is that there is strong multicollinearity between $x_1$, $x_2$ and $x_3$. We say multicollinearity is present in a regression model when two or more independent variables show a high correlation.
+
+### The Problem
+
+It affects the interpretability of a regression model since it compromises the statistical significance of independent variables. [Statistical significance](https://www.investopedia.com/terms/s/statistical-significance.asp) claims that an outcome resulting from testing or experimentation is unlikely to occur randomly. Instead, an occurrence can be attributed to a specific cause. Statistical significance reflects the confidence level of an outcome. Multicollinearity may make it difficult for us to distinguish between independent variables' isolated effects on a dependent variable.
+We should note that multicollinearity may not have a significant impact on the accuracy of the model. It has a greater impact on the reliability in finding out the effects of individual features of the model.
+
+### Causes
+
+There are many reasons why multicollinearity may occur. Let’s consider a handful. It may occur as a result of:
+
+1. The inclusion of identical variables. For example, one may have identical variables in a dataset, such as mass in kilograms and mass in pounds.
+
+2. Creation of new variables that are dependent on others. When we create variables that depend on other variables, we introduce redundant information to the model. As such, we may inadvertently encourage the occurrence of multicollinearity.
+
+3. Inadequate data. In some cases, when the data is inadequate, we may experience multicollinearity. This is due to the small sample size, which might, in turn, experience great variance.
+
+#### Variance Inflation Factor
+
+To learn the severity of multicollinearity, there are a few tests that may be carried out. We focus on the use of the variance inflation factor (VIF). The variance inflation factor represents the quotient of model variance with many terms against model variance with a single term. It is the score of an independent variable representing the degree to which other independent variables explain the variable. As such, it works out the magnitude of the correlation between independent variables.
```

Comment:
Can you explain the definition a bit more detailed manner. The definition is accurate for VIF, but I feel it needs more explanation in simpler words or through the help of an example.